doc/techReportChapters/gettingStarted.tex

```diff
@@ -171,3 +171,86 @@ \subsubsection{Testing Your Installation}
 \caption{\pgm{DeviceBrowser} Sample Display}
 \label{fig:deviceBrowser}
 \end{figure}
+
+\subsection{Running Real Designs - An Overview}
+This section will lead you through running an entire design from Vivado through
+RS2 and back.  In this example you will fully synthesize and implement a
+complete design in Vivado and then move it into RS2.  However, later as you gain
+experience you may choose to only synthesize  designs before exporting
+them to RS2 or you may choose to export and work with Vivado Out-Of-Context
+(OOC) Checkpoints.
+
+The steps to be covered include:
+
+\begin{enumerate}
+  \item Synthesize, place and route a design in Vivado
+  \item  Export the design from Vivado in the form of an RSCP (RapidSmith
+  Checkpoint)
+  \item Import the RSCP  into RS2
+  \item Run some analysis on the design
+  \item Export the design from RS2 as a TCP (Tincr Checkpoint)
+  \item Import the TCP back into Vivado
+\end{enumerate}
+
+In a real CAD flow you would likely do something more interesting
+than the analysis in Step 4 above, but this overview is intended to help you
+through the Vivado-to-RS2-back-to process so you can get started.
+
+Note: the entire flow requires the installation of Tincr as well as RS2 so if
+you skipped the Tincr installation above, go and install it now.
+
+\subsubsection{Synthesize, Place, and Route a Design in Vivado and Then Export
+It As a RSCP} If you desire, you may do this step any way you are used to using
+Vivado (with or without using the Vivado GUI for example).  In this step, we
+will do it using the Vivado Tcl interface.
+
+First, start up Vivado in command line mode.  This can be done using the
+``Vivado Tcl Shell'' program item or by executing ``vivado -mode tcl''.  Once
+the shell has started up, you may execute the following:
+                
+\vspace{-0.15in}  \begin{code}
```

Comment:
Done.  I see uses of the code environment and the lstlisting environment in the document.  What is your method for choosing where to use each?